modules/ossm-integrating-with-user-workload-monitoring.adoc

```diff
@@ -99,11 +150,11 @@ spec:
     interval: 30s
     relabelings:
     - action: replace
-      replacement: "<smcp_name>-<smcp_namespace>" # <2>
+      replacement: "basic-istio-system" # <2>
       targetLabel: mesh_id
 ----
 <1> Since {product-title} monitoring ignores the `namespaceSelector` spec in `ServiceMonitor` and `PodMonitor` objects, you must apply the `PodMonitor` object in all mesh namespaces, including the control plane namespace.
-<2> A shared Prometheus instance can include metrics from multiple meshes. However, if you are deploying Kiali, the metrics must be associated with a single mesh. To associate metrics with a single mesh, add a unique `mesh_id` to each one. Doing so narrows the query scope in Kiali to only the relevant mesh metrics.
+<2> The variable `"basic-istio-system"` is a combination of the SMCP name and namespace. It can be replaced with any other unique value within a cluster with multiple SMCPs. This is required only when Kiali is used and multiple SMCPs are deployed in a cluster. In case of cluster-wide deployment, it can be skipped. A shared Prometheus instance can include metrics from multiple meshes. However, if you are deploying Kiali, the metrics must be associated with a single mesh. To associate metrics with a single mesh, add a unique `mesh_id` to each one. Doing so narrows the query scope in Kiali to only the relevant mesh metrics.
```

Comment:
```suggestion
<2> Adding the `mesh_id` label can be skipped if there will never be more than one mesh being used with user workload monitoring on the cluster (be sure to also remove the `spec.prometheus.query_scope` from the Kiali deployment if following this guide). Otherwise, adding a `mesh_id` label is recommended as it allows for scoping metrics to their mesh of origin, and for Kiali it is required. The string `"basic-istio-system"` is a combination of the SMCP name and its namespace, but any label can be used as long as it is unique for every mesh using user workload monitoring in the cluster.
```

I took a stab at re-wording this to fix the following issues:

- For clarity, I think `"basic-istio-system"` should be called a "string" not a "variable".
- Additionally, the connection is never made between this value and the `spec.prometheus.query_scope` of the Kiali example resource given above, so it gives the impression that if I only have one mesh in my cluster I can just leave out this `mesh_id` relabeling, when I would also have to clear the `spec.prometheus.query_scope` of the Kiali resource.
- It also misses the nuance that the issue doesn't come from having more than one mesh _on the cluster_, in and of itself. There is only a need for the `mesh_id` label when there is more than one mesh being _who's metrics are being collected by user workload monitoring stack_. If including this detail makes the footnote too long or hard to follow, I don't think it's an important enough distinction to justify itself.

Reply:
Taking your suggestions via Slack of a shorter footnote, and then including the other information as a NOTE. I think that will be easier to follow than cramming it all into a footnote, or trying to separate it all into a different module, esp since it is all part of the same step.